Assets/Scripts/Player/PlayerLife.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class PlayerLife : MonoBehaviour
+{
+    public UnityEvent _died;
```

Comment:
https://trello.com/c/tJB8ZKOc/11-%D0%BE%D1%84%D0%BE%D1%80%D0%BC%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-unityevent